### PR TITLE
impr: upgrade ccd chart (FPLA-1620)

### PR DESCRIPTION
**Pull request by muyinatech**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-1620

### Change description ###
Upgrade CCD helm chart version to 4.2.3 and update values.preview template accordingly.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/fpl-case-service/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: fpl-case-service
 apiVersion: v1
 home: https://github.com/hmcts/fpl-ccd-configuration
-version: 1.10.11
+version: 1.10.12
```
Comment (tomaszpowroznik) [inline charts/fpl-case-service/Chart.yaml:4]: i would update minor version instead of patch

File: charts/fpl-case-service/values.yaml [hunk 2]
```diff
@@ -2,6 +2,9 @@ tags:
   fpl-ccd-stack: false
   ccd-idam-pr: false
```
Comment (tomaszpowroznik) [inline charts/fpl-case-service/values.yaml:3]: is ccd-idam-pr needed? is it not idam-pr now?

File: charts/fpl-case-service/values.yaml [hunk 3]
```diff
@@ -50,3 +53,6 @@ java:
         - AppInsightsInstrumentationKey
         - scheduler-db-password
   applicationPort: 4000
+
+ccd:
```
Comment (tomaszpowroznik) [inline charts/fpl-case-service/values.yaml:57]: tags:
  fpl-ccd-stack: false 
in first line should turn off ccd dependency, so this maybe is not needed

File: charts/fpl-case-service/values.yaml (hunk 2 above)
Comment (muyinatech) [reply charts/fpl-case-service/values.yaml:3]: Not need, so I will remove.

File: charts/fpl-case-service/Chart.yaml (hunk 1 above)
Comment (muyinatech) [reply charts/fpl-case-service/Chart.yaml:4]: Yup, I will change it.

File: charts/fpl-case-service/values.yaml (hunk 3 above)
Comment (muyinatech) [reply charts/fpl-case-service/values.yaml:57]: Agreed, so I will remove.

File: charts/fpl-case-service/values.yaml [hunk 4]
```diff
@@ -1,6 +1,8 @@
 tags:
   fpl-ccd-stack: false
-  ccd-idam-pr: false
+
+idam-pr:
```
Comment (tomaszpowroznik) [inline charts/fpl-case-service/values.yaml:4]: i would remove this one as 
tags:
 fpl-ccd-stack: false
should disable entire ccd deployment

**Pull request merged**